### DISCUSSION TITLE
NextJS SSR app router example

### DISCUSSION THREAD

**Discussion by nardokra** [jpudysz/react-native-unistyles#348] [Q&A]
How would you do the setup with NextJs app router? The document.js is not in place anymore. Can you maybe provide an example?

**Comment by jpudysz** [comment]
At the moment I don't have any new example.

**Comment by jpudysz** [comment]
We will publish new guide for 3.0

**Comment by ghost** [comment]
hi ...can you publish a nextjs example with v3 of unistyles?
